src/rotary-dialer-element.ts

```diff
@@ -0,0 +1,280 @@
+import { css, customElement, html, LitElement, property } from 'lit-element';
+
+enum Digit {
```

Comment:
How about just narrowing the `number` type? e.g.

`type Digit = 0 | 1 |  2 | 3 | 4 | 5 | 6 | 7 | 8 | 9`

Reply:
sure :) I know enums are heavy but I still tend to use them lol

Reply:
know I see I did it for the pattern matching (in the switch case) I'll try to tweak that if it will stay readable